src/scheduler/node_info.c

```diff
@@ -3912,6 +3952,64 @@ parse_selspec(char *select_spec)
 }
 
 /**
+ *	@brief compare two chunks for equality
+ *	@param[in] c1 - first chunk
+ * 	@param[in] c2 - second chunk
+ *
+ *	@return int
+ *	@retval 1 if chunks are equal
+ *	@retval 0 if chunks are not equal
+ */
+int compare_chunk(chunk *c1, chunk *c2) {
+	if (c1 == NULL && c2 == NULL)
+		return 1;
+	if (c1 == NULL || c2 == NULL)
+		return 0;
+	
+	if(strcmp(c1->str_chunk, c2->str_chunk) != 0)
```

Comment:
shouldn't we be doing a return '1' when strcmp matches? It seems that if the string matches completely then doing, compare_resource_req_list is redundant.
What do you think?